src/components/VerticalListMenu/examples/1.basic.jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import VerticalListMenuStateless from '../VerticalListMenu';
+import { buildStatefulComponent } from '../../../util/state-management';
+
+const VerticalListMenu = buildStatefulComponent(VerticalListMenuStateless);
+
+export default React.createClass({
+	render() {
+		return (
+			<VerticalListMenu style={{ width: '250px' }}>
```

Comment:
width could probably just take an integer of `250` rather than the px string. They are equivalent, but we should probably favor one over the other.